typescript/cloudwatch/evidently-client-side-evaluation-lambda/index.ts

```diff
@@ -0,0 +1,112 @@
+import * as appconfig from 'aws-cdk-lib/aws-appconfig'
+import * as cdk from 'aws-cdk-lib'
+import * as evidently from 'aws-cdk-lib/aws-evidently'
+import * as iam from 'aws-cdk-lib/aws-iam'
+import * as lambda from 'aws-cdk-lib/aws-lambda'
+import * as fs from'fs'
+
+const HIDE_FEATURE = 'hideFeature'
+const SHOW_FEATURE = 'showFeature'
+
+export class EvidentlyClientSideEvaluationLambdaStack extends cdk.Stack {
+  constructor(app: cdk.App, id: string) {
+    super(app, id);
+
+    // Create AppConfig resources
+    const application = new appconfig.CfnApplication(this,'AppConfigApplication', {
+      name: 'EvidentlyExampleProject'
+    });
+
+    const environment = new appconfig.CfnEnvironment(this, 'AppConfigEnvironment', {
+      applicationId: application.ref,
+      name: application.name + 'Environment'
+    });
+
+    // Create Evidently resources
+    const project = new evidently.CfnProject(this, 'EvidentlyProject', {
+      name: application.name,
+      appConfigResource: {
+        applicationId: application.ref,
+        environmentId: environment.ref
+      }
+    });
+
+    const feature = new evidently.CfnFeature(this, 'EvidentlyFeature', {
+      project: project.name,
+      name: 'MyExampleFeature',
+      variations: [
+        {
+          booleanValue: false,
+          variationName: HIDE_FEATURE
+        },
+        {
+          booleanValue: true,
+          variationName: SHOW_FEATURE
+        }
+      ]
+    })
+    feature.addDependsOn(project)
+
+    const launch = new evidently.CfnLaunch(this, 'EvidentlyLaunch', {
+      project: project.name,
+      name: 'ExampleFeatureLaunch',
+      executionStatus: {
+        status: 'START'
+      },
+      groups: [
+        {
+          feature: feature.name,
+          variation: HIDE_FEATURE,
+          groupName: HIDE_FEATURE
+        },
+        {
+          feature: feature.name,
+          variation: SHOW_FEATURE,
+          groupName: SHOW_FEATURE
+        }
+      ],
+      scheduledSplitsConfig: [{
+        startTime: '2022-01-01T00:00:00Z',
```

Comment:
How's this formatted? Can I say "Tomorrow"? (link to docs if you have to)

Reply:
will add docs